app/src/main/java/io/github/wztlei/wathub/ui/modules/Feedback/feedbacksendout.java

```diff
@@ -0,0 +1,24 @@
+package io.github.wztlei.wathub.ui.modules.Feedback;
+
+import retrofit2.Call;
+import retrofit2.http.Field;
+import retrofit2.http.FormUrlEncoded;
+import retrofit2.http.POST;
+
+// Rolf Li, July 2019
+
+public interface feedbacksendout {
```

Comment:
Please rename this interface/file to FeedbackInterface.java and place it in the folder  app/src/main/java/io/github/wztlei/wathub/net along with the other Retrofit-related interfaces

Reply:
Also, rename this folder from "Feedback" to "feedback" since Java package names should be lowercase